src/components/OptionsSelector.js

```diff
@@ -94,7 +98,15 @@ class OptionsSelector extends Component {
     }
 
     componentDidMount() {
-        this.textInput.focus();
+        if (!this.props.isLoading) {
+            this.textInput.focus();
+        }
```

Comment:
@Luke9389  Remove the check focus will be set to text input immediately, but the transition animation will not happen view will be presented immediately, receive input from the user immediately

Reply:
Sorry, I don't exactly understand why this change was made. Don't we want the focus to happen immediately?

Reply:
Since we use transition animation, setting `focus` to `textInput` immediately will stop the transition abruptly.